### PR TITLE
Generate Migrations

### LINKED ISSUE #3: [3.0pt] - Generate Migrations
Generate Migrations following ERD from #2 :
-  Create User:
  -  username, as string

- Create Experience:
  - name, as string
  - description, as text
  - experience_fee, as decimal
  - add_testing_fee, as decimal
  - reserve_full_table, as decimal
  - guests, as integer 
  
- Create Reservation: 
  - user as references
  - experience as references
  - date as, timestamp
  - city, as string

### PR DESCRIPTION
**Pull request by HFG43**
Hi @Dachrono Andy! I'm sending this pull request for you to review my code. Thanks in advance for your feedback!

Milestones 🎯:
📌 Generate Users, Experiences and Reservations Migrations following ERD
📌 Run Migration to create `schema.rb`

🎖️ General requirements:
Gitflow
No linter errors
Ensure the use of best practices and Ruby style

### CODE REVIEW

File: db/migrate/20231027212431_create_users.rb
Comment (Dachrono) [inline db/migrate/20231027212431_create_users.rb:1]: Please be kind and use "null: false" in your tables, this is a good way to cover issues with false data in your database

File: db/migrate/20231027212728_create_experiences.rb
Comment (Dachrono) [inline db/migrate/20231027212728_create_experiences.rb:1]: Please be kind and use "null: false" in your tables, this is a good way to cover issues with false data in your database

File: db/migrate/20231027212431_create_users.rb
Comment (Dachrono) [inline db/migrate/20231027212431_create_users.rb:1]: [OPTIONAL] You could use a full name to identify the username for every user

**Review by Dachrono — CHANGES_REQUESTED** [review]
# Hello Hernan! 👋 

Great Job I can see your effort 😄 👏 
# STATUS: CHANGES REQUIRED 
### To Highlight 💯

- Descriptive pr👍 
- Good and clean implementation👍 
- No error linters 👍

congrats

# Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20231027212728_create_experiences.rb
Comment (HFG43) [reply db/migrate/20231027212728_create_experiences.rb:1]: @Dachrono Thanks for this advice!

File: db/migrate/20231027212431_create_users.rb
Comment (HFG43) [reply db/migrate/20231027212431_create_users.rb:1]: @Dachrono Great! I have added name element

**Review by Dachrono — APPROVED** [review]
# Hello Hernan! 👋
 ## Congratulations! 🎉  

Great Job I can see your effort 😄 👏 You've done it. Your feature is complete!  There is nothing else to say  👍🏻 ✅  🥳 🎉 
# STATUS: APPROVED ✅ 🍾 
### To Highlight 💯

- Descriptive pr👍 
- Good and clean implementation👍 
- No error linters 👍
 
## Congratulations! 🎉 

congrats

# Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**